src/screens/Tag.js

```diff
@@ -0,0 +1,230 @@
+        </Property>
+        
+        <Property name="onRemove">
+          <Description>
+            If specified, the tag will include a remove button. This function
+            will be called when the remove button is activated. This should not
+            be used in conjunction with 'onClick'.
+          </Description>
+          <PropertyValue type="function">
+            <Example>{`() => {}`}</Example>
+          </PropertyValue>
+        </Property>
+        
+        <Property name="value">
+          <Description>Value text to place in the tag.</Description>
+          <PropertyValue type="string">
+            <Example>"value"</Example>
+          </PropertyValue>
+        </Property>
+      </Properties>
+
+      <ThemeDoc>
+        <Property name="tag.border">
+          <Description>Any box border settings.</Description>
+          <GenericBoolTrue />
+          <PropertyValue type="string">
+            <Example defaultValue>
+              {`{
+  xsmall: "1px",
+  small: "2px",
+  medium: "4px",
+  large: "12px",
+  xlarge: "24px",
+}`}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="tag.round">
+          <Description>How much to round the corners.</Description>
+          <GenericBool/>
+          <PropertyValue type="string">
+            <Example defaultValue>"large"</Example>
+            <SizesXsmallXlarge />
+            <Example>"full"</Example>
+            <Example>"any CSS size"</Example>
+          </PropertyValue>
+          <PropertyValue type="object">
+            <Example>{`{ size: "...", corner: "..." }`}</Example>
+            <PropOptions prop="corner">
+              <Example>"top"</Example>
+              <Example>"left"</Example>
+              <Example>"bottom"</Example>
+              <Example>"right"</Example>
+              <Example>"top-left"</Example>
+              <Example>"top-right"</Example>
+              <Example>"bottom-left"</Example>
+              <Example>"bottom-right"</Example>
+            </PropOptions>
+          </PropertyValue>
+        </Property>
+
+        <Property name="tag.name">
+          <Description>
+            An object specifying the properties for the name Text
+          </Description>
+          <PropertyValue type="object">
+            <Example defaultValue>
+              {`{}`}
+            </Example>
+            <Example>
+              {`{ weight: 600 }`}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="tag.pad">
+          <Description>
+            The amount of padding around the Box contents.
+          </Description>
+          <PropertyValue type="object">
+            <Example defaultValue>
+              {`{
+  horizontal: "small",
+  vertical: "xsmall",
+}`}
+            </Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="tag.remove">
+          <Description>
+            The properties on the remove button
```

Comment:
```suggestion
            Any valid Box props for the remove button.
```

Reply:
Aren't these Button props?

Reply:
Thanks, adjusted my wording. Pulled this phrasing from other components.